templates/taxi/manufacturer_list.html

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
     <h1>Manufacturer List
+      <a style="float: right" href="{%  url "taxi:manufacturer-create" %}">+</a>
     </h1>
```

Comment:
It's a good practice to use buttons for actions instead of links. I would recommend changing the '+' link to a button for better user experience and accessibility. Also, consider adding some text to this button for better accessibility, for example, 'Add new manufacturer'.